README.md

```diff
@@ -1,15 +1,32 @@
 # Java Echo Server
 [![Build Status](https://travis-ci.org/junehoe/java-echo-server.svg?branch=walking-skeleton)](https://travis-ci.org/junehoe/java-echo-server)
+
 An echo server written in Java.
 ## How to Run
 Clone the repository to your local machine, then go inside the directory by running:
 ```
 cd /java-echo-server
 ```
-Inside of the directory, run:
+Inside of the directory, run the following to start the server:
 ```
-gradle run
+gradle run -Pport=<desired-port-number>
 ```
+To run client:
+```
+nc <ip-address> <port-number-of-server-socket>
+```
+### Example
+If you would like to create a server on port `4242`, then you would run:
+```
+gradle run -Pport=4242
```

Comment:
I'm glad to see that the documentation is being kept up to date, during development.  This will be helpful to folks like me, who are just seeing the project for the first time.